fsspec/fuse.py

```diff
@@ -164,3 +213,122 @@ def run(fs, path, mount_point, foreground=True, threads=False):
             func()
         except KeyboardInterrupt:
             pass
+
+
+def mount(
+    url,
+    source_path,
+    mount_point,
+    log_file,
+    option,
+    foreground=True,
+    threads=False,
+    ready_file=False,
+):
+    kwargs = {}
+    for item in option:
+        key, value = item.split("=", 1)
+        if value.lower().endswith("[int]"):
+            value = int(value[: -len("[int]")])
+        elif value.lower().endswith("[bool]"):
+            value = value[: -len("[bool]")].lower() in ["1", "yes", "true"]
+
+        if "-" in key:
+            fs_name, setting_name = key.split("-", 1)
+            if fs_name in kwargs:
+                kwargs[fs_name][setting_name] = value
+            else:
+                kwargs[fs_name] = {setting_name: value}
+        else:
+            kwargs[key] = value
+
+    fs, url_path = url_to_fs(url, **kwargs)
+    logger.debug("Mounting %s to %s", url_path, mount_point)
+    run(fs, source_path, mount_point, log_filename=log_file, ready_file=ready_file)
+
+
+def main(args):
+    """Mount filesystem from chained URL to MOUNT_POINT.
```

Comment:
Include this text in the docs that get printed when no options (or "-h" option) passed?

Reply:
Options like `-h` still work. The `RawDescriptionArgumentParser` just uses the original `ArgumentParser.description` instead of formatted paragraph to avoid breaking command line examples in the docstring.